crates/vm/levm/src/vm.rs

```diff
@@ -255,6 +256,64 @@ impl VM {
             self.env.refunded_gas,
         );
 
+        if is_precompile(&current_call_frame.code_address) {
+            let precompile_result = execute_precompile(current_call_frame);
+
+            match precompile_result {
+                Ok((SUCCESS_FOR_RETURN, output)) => {
+                    self.call_frames.push(current_call_frame.clone());
+
+                    return Ok(TransactionReport {
+                        result: TxResult::Success,
+                        new_state: self.cache.clone(),
+                        gas_used: current_call_frame.gas_used.low_u64(),
+                        gas_refunded: self.env.refunded_gas.low_u64(), // There's no refunds in precompiles
```

Comment:
```suggestion
                        gas_refunded: 0,
```

Reply:
Done in [Set gas_refunded as zero](https://github.com/lambdaclass/ethrex/pull/1454/commits/bf5d5d1a6837420fa4e9c4268c6b6bf018b0abe8)